PYME/recipes/acquisition.py

```diff
@@ -0,0 +1,160 @@
+
+from .base import register_module, ModuleBase, OutputModule
+from .traits import Input, Output, CStr, Int, DictStrAny, Bool, Float, ListFloat
+import requests
+import json
+import numpy as np
+
+@register_module('UpdateSpoolerSettings')
+class UpdateSpoolerSettings(ModuleBase):
```

Comment:
This is a non-starter as a recipe module as it has side effects and is not a subclass of `OutputModule`. 

The solution we discussed had us passing any settings updated to the spool controller through the `StartSpooling` method in the action manager.

Reply:
I figured we'd cut it, but left it in until we discussed as I'd written/debugged it before we came to the conclusion we needed to change `SpoolController.StartSpooling` signature. Will take it out

Reply:
This is now removed.